modin/pandas/test/dataframe/test_iter.py

```diff
@@ -258,6 +258,21 @@ def test___setattr__():
     df_equals(modin_df, pandas_df)
 
 
+def test___setattr___getattr__():
+    # Use case from issue #4577
+    pandas_df = pandas.DataFrame([[1]], columns=["col0"])
+    modin_df = pd.DataFrame([[1]], columns=["col0"])
+
+    pandas_df.col0 = [3]
+    modin_df.col0 = [3]
+
+    df_equals(modin_df, pandas_df)
+    modin_df.col0 = pd.Series([3])
```

Comment:
```suggestion
    modin_df.col0 = pd.Series([5])
```

Let's use a new value just in case something goes wrong when assigning a `Series`.